sormas-api/src/main/java/de/symeda/sormas/api/RequestContextHolder.java

```diff
@@ -17,21 +17,24 @@
 
 public final class RequestContextHolder {
 
-	private static final ThreadLocal<RequestContextTO> requestContextHolder = new ThreadLocal<>();
+	private static final ThreadLocal<RequestContextTO> threadLocalContext = new ThreadLocal<>();
```

Comment:
A constant should be written in uppercase. I'm quite uncertain how this works, I hope it's not shared between invocations or users.

Reply:
Have a look at I18nProperties, it is the same. It is a constant but is set & cleaned up for each request.